kolibri/plugins/facility/assets/src/views/users/common/UsersTable.vue

```diff
@@ -149,6 +149,8 @@
             <KIconButton
               icon="optionsVertical"
               :disabled="!userCanBeEdited(content)"
+              :appearance="activeRowId === row[0].id ? 'raised-button' : 'flat-button'"
```

Comment:
The raised-button vs flat-button doesn't result in the appearance we expect here which is instead always a flat appearance. The issue is that we want to keep its active/hover state.

I think you'll just have to hard-code the same value as is defined in [the buttons mixin in KDS](https://github.com/learningequality/kolibri-design-system/blob/44d4cdff37be82354a01b5b91e81153423daabd9/lib/buttons-and-links/buttonMixin.js#L123) for this "flat" button on hover. That might not be the exact right color so be sure to check it to be sure